### PR TITLE
Match ember-cli conventions for file structure.

### PR DESCRIPTION
**Pull request by blimmer**
This addon morphed to an ember-cli addon, and didn't really follow the best practices of what the file structure of most ember addons look like.

This changeset does not affect any of the external facing APIs available to users.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "node": ">= 0.10.0"
   },
   "author": "",
-  "license": "Apache License 2.0",
+  "license": "Apache-2.0",
```
Comment (blimmer) [inline package.json]: Match SPDX-style licensing to quiet NPM warning.

**Comment by jamesarosen** [comment]
I don't object to this, but everything in `addon/` is intended to be private to the library. Are you requiring these things in your app?

**Comment by blimmer** [comment]
I'm not using anything in there, but I just started to have a look to fix #264 and noticed that the structure was very different from most other addons I've seen so I ran through and did this. If it's not what you had in mind, definitely feel free to just close this (the SPDX thing should happen either way and I'm happy to open a PR for that).

**Comment by jamesarosen** [comment]
Consistency is good for readability. I'm fine with the change. I restarted the beta build. I don't know why that was failing.

**Comment by blimmer** [comment]
I just updated to Ember CLI 1.13.1 to see if the upstream upgrades that come with 1.13.1 will help resolve the `beta` failures.

**Comment by blimmer** [comment]
Test failures due to [this issue](https://github.com/switchfly/ember-test-helpers/issues/59) and the new Ember 2.0.0-beta.3 just released this evening. Talked with @rwjblue in the Ember slack and he mentioned he was going to have a look at this tonight or tomorrow.

**Comment by jamesarosen** [comment]
Blocked by #278

**Comment by blimmer** [comment]
@jamesarosen ping on this.

**Comment by jamesarosen** [comment]
Still blocked. We need to get the test suite passing before we can merge anything.

**Comment by blimmer** [comment]
The tests on this passed. Did something else break?

**Comment by blimmer** [comment]
[build](https://travis-ci.org/jamesarosen/ember-i18n/builds/71766951)

**Comment by jamesarosen** [comment]
Master is broken against latest ember.

**Comment by rwjblue** [comment]
@blimmer - Can you rebase to confirm still passing?

File: bower.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
     "ember": ">=1.12.0 <1.13.0 || >=1.13.1 <3.0.0",
     "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
     "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.4",
-    "ember-qunit": "0.3.3",
+    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.5",
+    "ember-qunit": "0.4.2",
```
Comment (rwjblue) [inline bower.json]: Might as well update to 0.4.6 while we are at it :dancer:

**Comment by rwjblue** [comment]
@jamesarosen - You mention that master is broken against beta and canary, but that is precisely what this PR fixed when submitted.  We should get it updated to ember-qunit@0.4.6 and rebased to make sure that things are still passing, but it likely is...

**Comment by kellyselden** [comment]
@jamesarosen @rwjblue Confirmed that updating qunit references fixes the tests in beta and canary.

**Comment by kellyselden** [comment]
added a PR with just the qunit update #285

**Comment by blimmer** [comment]
Just let this run again and it's definitely fixed here. I'd prefer to merge this one over #285.

**Comment by jamesarosen** [comment]
> but that is precisely what this PR fixed when submitted

I saw a failure on this PR this morning, but that might have been a random fluke. I didn't investigate thoroughly enough.

**Comment by jamesarosen** [comment]
+1

**Pull request merged**